src/main/java/com/lenis0012/bukkit/loginsecurity/util/LoggingFilter.java

```diff
@@ -25,26 +30,37 @@ private Result handle(String message) {
             }
         }
 
+        if(loginSecurityConfig.isUseCommandShortcut()) {
+            final String loginCommandShortcut = loginSecurityConfig.getLoginCommandShortcut();
+            final String registerShortcut = loginSecurityConfig.getRegisterCommandShortcut();
+            if(message.startsWith(loginCommandShortcut)
```

Comment:
I believe that the shortcut commands don't have a space at the end. That would mean that the default of "/l" would block any command from being logged that starts with "l". That probably isn't really desirable.

This can be prevented by appending a space to the shortcuts.

Reply:
Try adding a test case for the command "/lwc help" or something.

Reply:
Should these also have a space at the end?
https://github.com/lenis0012/LoginSecurity/blob/67695d4ecbd66de0cc3f4768f7576a81aede4468/src/main/java/com/lenis0012/bukkit/loginsecurity/util/LoggingFilter.java#L14

Reply:
that is not really needed because those commands are more specific.